Multisig/App/NSNotification+Events.swift

```diff
@@ -14,6 +14,9 @@ extension NSNotification.Name {
     static let selectedSafeChanged = NSNotification.Name("io.gnosis.safe.selectedSafeChanged")
     static let selectedSafeUpdated = NSNotification.Name("io.gnosis.safe.selectedSafeUpdated")
     
+
+    static let safeCreated = NSNotification.Name("io.gnosis.safe.safeCreated")
+    static let showBalances = NSNotification.Name("io.gnosis.safe.showBalances")
```

Comment:
these two notifications were deleted in the main branch. please use the `safeCreationUpdate`